docs/pages/guides/authentication.mdx

```diff
@@ -58,8 +58,8 @@ If you'd like to see more, you can [open a PR](https://github.com/expo/expo/edit
   <GridItem title="GitHub" protocol={['OAuth 2']} image={ASSETS.github} />
   <GridItem
     title="Google"
-    protocol={['OAuth 2', 'OpenID']}
-    href="/guides/google-authentication/"
+  protocol = { ['OAuth 2', 'OpenID']}
+  href ="/guides/google-authentication/"
```

Comment:
```suggestion
    protocol={['OAuth 2', 'OpenID']}
    href="/guides/google-authentication/"
```

As per our Prettier configuration, this formatting is incorrect. Let's revert it.